pytorch_blade/src/compiler/jit/tool_funcs.cpp

```diff
@@ -38,13 +38,27 @@ torch::jit::Module clone_cpp_module(torch::jit::Module& self) {
   return torch::jit::Module(self.clone(false));
```

Comment:
TorchScript force exportable methods must have a single return value, otherwise one would run into exceptions when saving the Module.

https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/serialization/python_print.cpp#L778-L782

Reply:
Any other functions deps on this function? If not, maybe we can move it to `debug_utils.h`, and rename `debug_utils` to `graph_dumper` which is a more accurate name.

And maybe we need a ut on the dumper tools instead of just `return_multi_outputs_with_tuple ` .

Reply:
Yes, this function is used in TorchBlade early.